src/include/ca-management/new_cert_read_write.rb

```diff
@@ -1932,7 +1933,7 @@ def create_default_CA_certificate
             _(
               "The default certificate can also be created in\nthe CA Management module.\n"
             )
-          )
+          ) unless Mode.autoinst
```

Comment:
Should it be skipped also during `autoupgrade`? In that case, maybe you would like to use `Mode.auto`. If no, just ignore this comment :)